package.json

```diff
@@ -43,6 +43,13 @@
     "typescript": "4.1.3"
   },
   "peerDependencies": {
-    "eslint": "^7.5.0"
+    "eslint": "^7.5.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.3.4",
+    "eslint-plugin-jsdoc": "^32.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.21.0",
+    "@typescript-eslint/parser": "^4.21.0",
+    "@angular-eslint/template-parser": "^2.0.2"
```

Comment:
On our project we are using `v12.5.0` for `@angular-eslint/template-parser`. I cannot install this library without `--force` with current `peerDependencies`

Should we check and update all of the dependencies to the latest major version?

Reply:
That would cause issues in the ngx-eds lib. I would suggest we change this to "^2.0.2 | v12.5.0"

Reply:
Either that, or we just allow whole range  larger than 2 and smaller than 13?

Reply:
Changed to `"@angular-eslint/template-parser": ">=2.0.2 <13.0.0" `

I actually thought they went from 2 to 12 but there are also versions 3 and 4 so your range suggestion will be the best choice.